lms/static/js/jwt_auth/utils.js

```diff
@@ -0,0 +1,111 @@
+/**
+ * Utils file to support JWT Token Authentication.
+ *
+ * Temporarily copied from the edx/frontend-platform
```

Comment:
"Temporary" caught my eye. Is there any follow up work we need to do here? If so, is it captured in a ticket in the backlog anywhere?

Reply:
HAH! Disregard. I was reviewing this PR from the bottom to the top. I see we added that README when we pulled this code in. So it sounds like this is taken care of =D

Reply:
@staubina

Reply:
Yup. Please see the README.